### PR TITLE
Typos fixes, minor content edits

### PR DESCRIPTION
**Pull request by jasonbraganza**
This was wonderfully done!

Schuby, you’re awesome!

Please check and commit if you folks are ok with the fixes

### CODE REVIEW

File: issues/2018-03-19/issue_2018-03-19.md [hunk 1]
```diff
@@ -5,42 +5,48 @@
...
+Who knows, you might find yourself in a situation where you have the possibility to contribute sooner than you think.
 
-It's encouraging to see how core developers take care, make suggestions and guide contribuors to deliver a professional fix. The things you learn and experience during a process like this are priceless.
+Here’s a recent example. Diceware is a convenient tool to create strong random passwords. Bhavin found an uncaught exception,  and decided to commit a fix.
 
-A very much worth reading blog post for everyone who's interested in how open source realley works. 
+To write the code and create a pull request is however just the start. Read [Bhavin's blog post about this pull request](https://geeksocket.in/blog/contributing-diceware-tool/) for all details about the exciting progress from discovering a bug to get the own code merged into the project, making it a little better.
```
Comment (SaptakS) [inline issues/2018-03-19/issue_2018-03-19.md]: I feel `to getting the code fix merged` makes more sense. @jasonbraganza views?

Comment (jasonbraganza) [reply issues/2018-03-19/issue_2018-03-19.md]: Another typo I missed. I’d rather have it be ```their own code merged````
Gives the contributor (Bhavin) a better sense of ownership

Comment (jasonbraganza) [reply issues/2018-03-19/issue_2018-03-19.md]: Another typo I missed. I’d rather have it be ```their own code merged```
Gives the contributor (Bhavin) a better sense of ownership

Comment (SaptakS) [reply issues/2018-03-19/issue_2018-03-19.md]: Once you push that change, I think it's good to be merged.

File: issues/2018-03-19/issue_2018-03-19.md [hunk 2]
```diff
@@ -5,42 +5,48 @@
...
-- [kushaldas - https://githubengineering.com/ddos-incident-report/](https://githubengineering.com/ddos-incident-report/): In February, github has been unavailable for a couple of minutes due to a distributed denial-of-service (DDoS) attack. Read about the background and the actions taken.
-- [kushaldas - https://rfc.dgplug.org/](https://rfc.dgplug.org/): RFCs, the Requests for Comments form the guidelines provided by the Internet Engineering Task Force, that our internet is built on. Did you know dgplug is hosting a copy?
+- [bhavin192 - https://blog.cloudflare.com/using-go-as-a-scripting-language-in-linux/](https://blog.cloudflare.com/using-go-as-a-scripting-language-in-linux/): Go’s getting more and more popular by the day, being relatively easy to learn and allowing rapid development. This post describes how use to Go as a scripting language by make .go files executable without the need of compiling/installing the code first.
```
Comment (ghost) [inline issues/2018-03-19/issue_2018-03-19.md]: I think it should say `how to use Go as a scripting language and make .go files`

the `by make .go files` was actually another mistake that I make ;)

**Review by ghost — CHANGES_REQUESTED** [review]
Thanks for the review @jasonbraganza, that was quick! I'm happy with the changes, could you update the two commented items so the changes can be merged?

**Review by jasonbraganza — COMMENTED** [review]
a-ok! now

**Pull request merged**